book/text.md

```diff
@@ -915,8 +909,7 @@ logic itself in a new `get_font` function:
 def get_font(size, weight, style):
     key = (size, weight, style)
     if key not in FONTS:
-        font = tkinter.font.Font(size=size, weight=weight,
-            slant=style)
+        font = tkinter.font.Font(size=size, weight=weight, slant=style)
```

Comment:
The line wrapping is not needed, we have enough space.

Reply:
Update: this line is otherwise 72 characters, so maybe wrapping is indeed needed? But in the book proof there is clearly enough space that we don't need to wrap.

Reply:
This is what's causing tests to fail. I suggest we bump the minimum line width to something like 75 in the checker and suggest the change. But I won't insist on it, it doesn't matter that much.

Reply:
I'd prefer to leave the code as-is and not mess around with line limits, this doesn't look critical to me.